### PR TITLE
Added docker compose file to use artifact registry and new deployment

### PR DESCRIPTION
**Pull request by jnsproul**

### CODE REVIEW

File: docker-compose-artifact-registry.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: "3.8"
```
Comment (cbcunc) [inline docker-compose-artifact-registry.yml]:
```suggestion
```

File: docker-compose-artifact-registry.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (cbcunc) [inline docker-compose-artifact-registry.yml]:
```suggestion
```

File: docker-compose-artifact-registry.yml (hunk 1 above)
Comment (cbcunc) [reply docker-compose-artifact-registry.yml]: `version` has been deprecated.

File: docker-compose-artifact-registry.yml
Comment (cbcunc) [inline docker-compose-artifact-registry.yml:1]: Have you tested this? it looks fine except for the minor shakes I suggested.

**Review by cbcunc — CHANGES_REQUESTED** [review]

**Comment by cbcunc** [comment]
We'll probably want to make the URI for AR an environment variable later. But just leave it be for now.

File: docker-compose-artifact-registry.yml
Comment (jnsproul) [reply docker-compose-artifact-registry.yml:1]: Tested

**Comment by jnsproul** [comment]
All requested changes have been made this can go in whenever

**Pull request merged**